platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/cache/CookieCache.java

```diff
@@ -58,9 +58,7 @@ public void addActiveSp(String key, String activeSp) {
             LOGGER.error("Cannot add sp [{}] for [{}]: does not exist", activeSp, key);
             return;
         }
-        synchronized (dataWrapper) {
```

Comment:
In fact, on reviewing this class further I think that the `getSamlAssertion` and `removeSamlAssertion` methods should also contain sync blocks on the retrieved `dataWrapper`. @rzwiefel, what are your thoughts on that?